src/features/dashboard/components/Table/__tests__/Table.test.tsx

```diff
@@ -131,11 +136,15 @@ describe('Table', () => {
   });
 
   it('Should delete token on delete button clicked', async () => {
-    const lastusedCells = screen.getAllByTestId('lastused-cell');
-    const firstCell = lastusedCells[0];
-    const withinFirstCell = within(firstCell);
-    const deleteButton = withinFirstCell.getByRole('button');
-
+    const actionCells = await screen.findAllByTestId('token-action-cell');
+    const firstActionCell = actionCells[0];
+    const withinActionCell = within(firstActionCell);
+
+<<<<<<< HEAD
+    const openDeleteDialogButton = withinActionCell.getByTestId('delete-token-button');
+    await userEvent.click(openDeleteDialogButton);
+    expect(openDeleteDialogButton).toBeInTheDocument();
+=======
```

Comment:
I think the syntax is wrong here.

Reply:
Fixed this as well, following how it was done in the apps table test case.